### DISCUSSION TITLE
Support

### DISCUSSION THREAD

**Discussion by amberatwood** [community/community#61870] [New to GitHub]
### Select Topic Area

Show & Tell

### Body

I am new any good pointers? 💓

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @amberatwood , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category and are explicit about what it entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. 

But also allow me to say welcome to GitHub and the GitHub Community 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
